docs/spec-template.md

```diff
@@ -3,10 +3,10 @@ id: spec-template
 title: Template Structure
 ---
 
-An Accord Project template is composed of three elements: 
+An Accord Project template is composed of three elements:
 
 - Natural Language, the grammar for the legal text of the template
-- Model, the data model that backs the template 
+- Model, the data model that backs the template
 - Logic, the executable business logic for the template
 
 ![Cicero Template](assets/template.png)
```

Comment:
If we are changing the formatting to `{{` and `}}`, we should update the diagram too.